commit 26fe1a01f8f01ca58c5c4382fb12de4e93cce4f1
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Feb 4 12:45:25 2019 +0000

    configs: Add VIDEO_BCM2835 to bcmrpi3_defconfig
    
    This is now shown to work with 64 bit kernels, so add it to
    the defconfig.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 0fca7f9ef0af..80691b1048ad 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -1070,6 +1070,7 @@ CONFIG_FB_TFT_WATTEROTT=m
 CONFIG_FB_FLEX=m
 CONFIG_FB_TFT_FBTFT_DEVICE=m
 CONFIG_SND_BCM2835=m
+CONFIG_VIDEO_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1498_26fe1a01f8f01ca58c5c4382fb12de4e93cce4f1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1498_26fe1a01f8f01ca58c5c4382fb12de4e93cce4f1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
